src/main/java/seedu/address/model/person/IsInFilterPredicate.java

```diff
@@ -0,0 +1,47 @@
+package seedu.address.model.person;
+
+import java.util.Set;
+import java.util.function.Predicate;
+
+/**
+ * Tests that a {@code Person}'s {@code CategoryCode, Rating} matches the specified {@code CategoryCode, Rating}.
+ */
+public class IsInFilterPredicate implements Predicate<Person> {
```

Comment:
Everything looks good to me, except for a pretty small thing about naming (`IsFilterablePredicate` instead of `IsInFilterPredicate` since Filter is not an object

Reply:
okay got it!